### ISSUE TITLE
Password recovery configuration error - SSL Require server authentication

### ISSUE THREAD

**Issue by nicomon24** [baasbox/baasbox#937]
Some email servers requires SSL server authentication (Aruba.it) , but it is always set to false:

File baasbox/app/com/baasbox/service/user/UserService.java
Line 634
`email.setSSLCheckServerIdentity(false);`

Please add the possibility to set this value in the settings

Nico

**Comment by giastfader** [comment]
Have you tried to set the network.smtp.ssl and network.smtp.tls settings?

**Comment by nicomon24** [comment]
Yes I've tried it, no result! So I switched to a gmail account and it works. 
The option is specifically required by Aruba provider
